source/assets/js/components/fef-sticky-header.js

```diff
@@ -1,126 +1,47 @@
-            this.registerListeners();
-        }
-    }
+        this.initializeAffix();
 
-    registerListeners() {
-        $(window).on('scroll', FefDebounceHelper.debounce(() => this.afterScrolling(), DEBOUNCE_TIME) );
-        $(window).on('resize', FefDebounceHelper.debounce(() => this.afterResize(), DEBOUNCE_TIME) );
+        $(window).on('resize.sticky-header', FefDebounceHelper.debounce(() => this.reInitializeAffix(), DEBOUNCE_TIME));
     }
 
-    afterScrolling() {
-        let scrollTop = $(window).scrollTop();
-        let that = this;
-        let scrollDifference = Math.abs(this.lastScrollTop - scrollTop);
-
-        /*
-         * Sticky Header must stay as it is after resizing
-         * Sticky Header may not change state after a minimal scrolling (5 pixel)
-         * Sticky Header may not change state if scrollTop has a negative value (safari)
-         */
-        if (!this.hasResized && scrollDifference > 5 || scrollTop < 0) {
-            // scroll up > show full header
-            if (scrollTop <= this.lastScrollTop) {
-
-                this.$stickyContainer.css('margin-top', '0');
-                this.$stickyContainer.addClass('js-sticky-container--full');
-                this.$masthead[0].className = this.$masthead[0].className.replace(/\-\-off\-theme\-/g, '--theme-');
-
-                if (this.scrollDirection !== 'up') {
-                    this.$subnav.addClass('subnav--in-transition');
-                    setTimeout(
-                        function () {
-                            that.$subnav.removeClass('subnav--in-transition');
-                        },
-                        200
-                    );
-                }
-
-                this.scrollDirection = 'up';
-            }
-            // scroll down > show small header
-            else {
-
-                $('.affix').css('margin-top', '-' + this.affixMarginTop + 'px');
-                this.$stickyContainer.removeClass('js-sticky-container--full');
-
-                if(scrollTop >= this.affixMarginTop && this.scrollDirection !== 'down') {
-                    this.$subnav.addClass('subnav--in-transition');
-                    this.$masthead.addClass('masthead--in-transition');
-                    setTimeout(
-                        function() {
-                            that.$subnav.removeClass('subnav--in-transition');
-                            that.$masthead.removeClass('masthead--in-transition');
-                            that.$masthead[0].className = that.$masthead[0].className.replace(/\-\-theme\-/g, '--off-theme-');
-                        },
-                        200
-                    );
-                    this.scrollDirection = 'down';
-                }
-            }
+    initializeAffix() {
+        // Do not initialize in case of home landingpage on a breakpoint larger then smartphone
+        if(!(!FefResponsiveHelper.isSmartphone() && this.$masthead.hasClass('masthead--home'))) {
```

Comment:
Würde hier die doppelte Verneinung rausnehmen --> only initialize on Smartphone and if not on home

if(isSmartphone && !hasClass) ...

Reply:
es soll aber immer initialisieren, ausser auf tablet/desktop on home...
wir haben kein isTabletUp o.ä. in der js/css-bridge, deshalb müsste es sonst heissen:
(!((isTablet() || isDesktop || isDesktopWide) && hasClass))...

ich lass' es drum, ok?